test/cpp/BUILD

```diff
@@ -147,6 +147,7 @@ ptxla_cc_test(
             "//torch_xla/csrc:tensor",
             "@com_google_googletest//:gtest_main",
             "@org_tensorflow//tensorflow/compiler/xla:permutation_util",
+            "@org_tensorflow//tensorflow/compiler/xla/stream_executor:cuda_platform",
```

Comment:
What is this dependency for? Should it be in a runtime build target?

Reply:
this dep is included before apply OpenXLA's patch in `xla_bzl.diff`, error log shows missing this dep when build, will try again without this modify, we might have this dep from xla_cc_test now, thanks, Will

Reply:
updated in https://github.com/pytorch/xla/pull/5165/commits/fd2d9aa2ce84e5066eb1870318578f959af19b22

Reply:
this dep is needed because of https://github.com/pytorch/xla/actions/runs/5284848153/jobs/9562988206?pr=5165